src/main/docker/docker-compose.yml

```
@@ -0,0 +1,28 @@
+version: '3.3'
+services:
+  app:
+    image: 'docker-spring-boot-postgres:latest'
+    build:
+      context: .
```

Once you moved the `docker-compose.yml` from the current location.

```suggestion
      context: .
      dockerfile: src/main/docker/Dockerfile
```

You could incorporate the argument `JAR_FILE` from the current state of the `Dockerfile` (see  https://docs.docker.com/compose/compose-file/build/).